packages/x-telemetry/src/runtime/sender.ts

```diff
@@ -0,0 +1,72 @@
+import type { TelemetryContextType } from './get-context';
+import { getTelemetryEnvConfigValue } from './config';
+import { TelemetryEvent } from '../types';
+import { fetchWithRetry } from './fetcher';
+import * as packageJson from '../../package.json';
```

Comment:
```suggestion
import * as packageJson from '../../package.json';
```

maybe this would fix the warning (in our docs, and likely in a lot of other project's).

<img width="1302" alt="SCR-20250302-btck" src="https://github.com/user-attachments/assets/331b6a54-be74-4f84-89da-d69744410591" />

Otherwise, we can use Babel like we did for `__RELEASE_INFO__`. You can search for `MUI_MAJOR_VERSION`, similar work, MUI X might need to expose this in other instances: https://github.com/mui/material-ui/pull/43190.

Reply:
Done in #16931